### DISCUSSION TITLE
Mason does not understand that installation completed

### DISCUSSION THREAD

**Discussion by debasish-raychawdhuri** [mason-org/mason.nvim#1308] [Q&A]
Hi,

I am using Mason to install language servers. Mason installation process never shows completion and no lsp is ever being installed.

I am a noob, so please let me know how to find diagnostics and such.

**Comment by williamboman** [comment]
Hello! What do you mean by "installation process never shows completion"? You can find logs by doing `:MasonLog` (also see `:h mason-debugging` for how to control log levels).
